src/main/java/com/google/cloud/resourcemanager/spi/v1beta1/HttpResourceManagerRpc.java

```diff
@@ -301,4 +311,66 @@ public Policy replacePolicy(String projectId, Policy newPolicy) throws ResourceM
       throw translate(ex);
     }
   }
+
+  @Override
+  public void clearOrgPolicy(String resource, ClearOrgPolicyRequest request) {
+    try {
+      resourceManager.folders().clearOrgPolicy(resource, request).execute();
+    } catch (IOException ex) {
+      throw translate(ex);
+    }
+  }
+
+  @Override
+  public OrgPolicy getEffectiveOrgPolicy(String resource, GetEffectiveOrgPolicyRequest request) {
+    try {
+      return resourceManager.folders().getEffectiveOrgPolicy(resource, request).execute();
+    } catch (IOException ex) {
+      throw translate(ex);
+    }
+  }
+
+  @Override
+  public OrgPolicy getOrgPolicy(String resource, GetOrgPolicyRequest request) {
+    try {
+      return resourceManager.folders().getOrgPolicy(resource, request).execute();
+    } catch (IOException ex) {
```

Comment:
IOExceptions should not be converted into runtime exceptions

Reply:
done